akka-persistence-tck/src/main/scala/akka/persistence/CapabilityFlags.scala

```diff
@@ -52,3 +52,9 @@ trait SnapshotStoreCapabilityFlags extends CapabilityFlags {
   // no flags currently
 }
 //#snapshot-store-flags
+
+trait CurrentEventsByTagCapabilityFlags extends CapabilityFlags {
```

Comment:
I have a question about this one. Some journals may not be able to sort on creationdate eg. I believe that cassandra could have an issue with this. Those storage engines store events per persistenceId and thus can only return a (sorted) persistenceId/event stream. LevelDB and JDBC can return an event stream sorted by creationdate because the data is stored on a single table. Should we use the compatibilityflag so that the query api implementation can select which strategy it uses? I mean the `supportsOrderingByDateIndependentlyOfPersistenceId` or `supportsOrderingByPersistenceIdAndSequenceNr` of the class `CurrentEventsByTagCapabilityFlags` and of course (but this one doesn't exist yet) the `EventsByTagCapabilityFlags` When enabled these tests will be run